### ISSUE TITLE
ZOMBIE vnodes need a way to hand off data

### ISSUE THREAD

**Issue by ghost** [Factual/skuld#4]
Skuld.politics should be responsible for periodically asking ZOMBIE vnodes to ensure their data is safely handed off to the current cohort, and shutting down. Not sure exactly how this should work yet, but I think we need permission from a newer epoch leader to do it. Maybe that leader should coordinate the handoff?

**Comment by eric** [comment]
I'm having trouble seeing what needs to be done here.

The election docs talk about something after step 8 that is moving data off of zombie nodes, but I don't see anything in the code doing that.

Also, it seems that as part of an election, all of the claims on a zombie node should be pulled off, so we should be fine, anyways.

**Comment by aphyr** [comment]
Yeah. Zombie vnodes need to enforce that their data is all immutable, then attempt to merge each existing record into at least QUORUM active nodes before deleting it locally. Only once all local data is deleted can the vnode shut down.

**Comment by aphyr** [comment]
(if you can't merge your data into a majority of new vnodes, we could lose data by dropping the vnode prematurely)
